Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rust:1.77.0 as builder
+LABEL Description="OpenLan CGW (Build) environment"
+
+WORKDIR /usr/src/openlan-cgw
+
+RUN rustup target add x86_64-unknown-linux-gnu
+
+RUN apt-get update -q -y  && apt-get install -q -y \
+	pkg-config \
+	build-essential \
+	cmake \
+	protobuf-compiler \
+	libssl-dev
+
+CMD ["make", "-C", "/usr/src/openlan-cgw", "cgw-app"]
+
+FROM busybox:glibc as cgw-img
```

Comment:
Why use `busybox` as base image?
Why not use `gcr.io/distroless/static-debian12` for example. It is almost 4 times smaller and distroless, i.e. more secure.

Reply:
i agree here

Reply:
> Why use busybox as base image?
As of now we need at least some enviroment and apps in development process - thus we need some base docker img.
Busybox is small (2 to 5 - worst case -  megs of data) and we don't need some specific base; scratch will be used in the future; but as of now i don't see any reasons to switch busybox;

As for the size - current img size is ~400 megs, it's due to massive amount of /usr/lib copied into resulting image. We can improve this in the future, as now we're concentrated on the rust code base itself to enrich the app with features